modules/installation-mirror-repository.adoc

```diff
@@ -235,18 +206,12 @@ $ oc adm release extract -a ${LOCAL_SECRET_JSON} --command=openshift-install "${
 ----
 endif::[]
 +
-[IMPORTANT]
+[NOTE]
 ====
-To ensure that you use the correct images for the version of {product-title}
-that you selected, you must extract the installation program from the mirrored
-content.
-
 You must perform this step on a machine with an active internet connection.
-
-If you are in a disconnected environment, use the `--image` flag as part of must-gather and point to the payload image.
 ====
-+
-. For clusters using installer-provisioned infrastructure, run the following command:
+
+. For clusters on {rh-virtualization-first} that use installer-provisioned infrastructure, run the following command:
```

Comment:
RHV? and this section is talking about mirror procedure, no need to mention openshift-install running.